pkg/new-kernel/kernel_config-5.6.x-aarch64

```diff
@@ -4832,17 +4832,18 @@ CONFIG_VIRTIO_MMIO=y
 # Xen driver support
 #
 CONFIG_XEN_BALLOON=y
-# CONFIG_XEN_BALLOON_MEMORY_HOTPLUG is not set
+CONFIG_XEN_BALLOON_MEMORY_HOTPLUG=y
```

Comment:
Are we planning on using balooning?

Reply:
I see it is set the same way in the 4.X config files, so this is nothing new. Ignore comment.